projects/demo-maps/src/app/route-components/route-example-threejs/route-example-threejs.component.html

```diff
@@ -0,0 +1,49 @@
+<main class="content-area">
```

Comment:
Please adjust the columns so it is better usable on smaller screens. (https://clarity.design/documentation/grid)
Two maps side by side is always a hard to layout but a lot of scrollbars look really ugly.

Reply:
Done!